File: BlackJack.BusinessLogic/GameLogic/PlayerLogic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BlackJack.BusinessLogic.GameLogic
{
    public class PlayerLogic
    {
        public string Name;
        public bool IsBot;
        public List<CardLogic> Hand = new List<CardLogic>();
    }
}

File: BlackJack.Corang/ClientApp/app/models/playerLogic.ts
import { CardLogic } from "./cardLogic";

export interface PlayerLogic {
    id: string;
    currentRoundPlayerId: string;
    name: string;
    isBot: boolean; 
    chips: number;
    bet: number;

    cards: CardLogic[];
    score: number;
    x: number;
    y: number;
}
